spec/features/user_creates_review_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+
+feature 'user creates review', %{
+  As a signed up user
+  I want to create new review
+  So that other users can see what I think of the item
+} do
+  scenario 'creates review succcessfully' do
+    user = FactoryGirl.create(:user)
+    category = FactoryGirl.create(:category)
+    item = FactoryGirl.create(:item, user: user, category: category)
```

Comment:
Why not `FactoryGirl.create(:item)` and then use the associated `user` and `category` instead of creating all three?